src/examples/simple_zkapp.mjs

```diff
@@ -46,31 +38,26 @@ let account1 = Local.testAccounts[0].privateKey;
 let zkappKey = PrivateKey.random();
 let zkappAddress = zkappKey.toPublicKey();
 
-let initialBalance = 10_000_000_000;
 let initialState = Field(1);
+let zkapp = new SimpleZkapp(zkappAddress);
+
+console.log('compile');
+SimpleZkapp.compile(zkappAddress);
```

Comment:
If you didn't call this, down below the following error would be thrown:
`Error: Cannot prove execution of update(), no prover found. Try calling `await SimpleZkapp.compile()` first, this will cache provers in the background.`

Also, the deploy transaction would silently have now verification key in it (which wouldn't matter, because the mock interface doesn't require one)

One idea I had was to introduce a parameter on the `Mina` implementations, called `proofLevel` or similar, that would determine whether actual proofs and verification keys were required or not

Reply:
Re other questions: compile takes ages, and blocks the main thread if you do it like here

Compiling automatically whenever it's needed makes sense to me -- but I would possibly log some warning that it was triggered implicitly, so that devs don't think snarkyjs hangs for no reason